### PR TITLE
Fixed CI so it is green

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: src/Core/composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
         "psr/cache": "^1.0",
         "psr/log": "^1.0",
         "symfony/deprecation-contracts": "^2.1",
-        "symfony/http-client": "^4.4 || ^5.0, !=5.2.0",
+        "symfony/http-client": "^4.4 || ^5.0,!=5.2.0",
```
Comment (jderusse) [inline src/Core/composer.json:21]: For consistency with spaces arround ` || `.

File: src/Service/Ecr/tests/Unit/Input/GetAuthorizationTokenRequestTest.php [hunk 2]
```diff
@@ -9,17 +9,15 @@ class GetAuthorizationTokenRequestTest extends TestCase
 {
     public function testRequest(): void
     {
-        $input = new GetAuthorizationTokenRequest([
-            'registryIds' => ['000000000000'],
-        ]);
+        $input = new GetAuthorizationTokenRequest();
```
Comment (jderusse) [inline src/Service/Ecr/tests/Unit/Input/GetAuthorizationTokenRequestTest.php:12]: :+1:

**Review by jderusse — APPROVED** [review]
LGTM except the consistency in operator separation in composer.json

File: src/Core/composer.json (hunk 1 above)
Comment (Nyholm) [reply src/Core/composer.json:21]: That will fail `composer normalize`

https://github.com/async-aws/aws/pull/896/checks?check_run_id=1531089092#step:4:31

Comment (jderusse) [reply src/Core/composer.json:21]: oh... ok then

File: src/Integration/Flysystem/S3/composer.json [hunk 3]
```diff
@@ -22,10 +22,10 @@
         "php": "^7.2.5 || ^8.0",
         "async-aws/s3": "^1.5",
         "league/flysystem": "^1.0.65",
-        "league/mime-type-detection": "^1.0"
+        "league/mime-type-detection": "^1.5.1"
```
Comment (jderusse) [inline src/Integration/Flysystem/S3/composer.json:25]: really? We are not compatible with previous version?

Comment (Nyholm) [reply src/Integration/Flysystem/S3/composer.json:25]: I just bumped it to the latest. We are compatible with 1.0. Should I revert?

Comment (jderusse) [reply src/Integration/Flysystem/S3/composer.json:25]: Yes please, If we are compatible with 1.0, we should force user to upgrade to 1.5.1

Comment (Nyholm) [reply src/Integration/Flysystem/S3/composer.json]: Okey. Thank you

**Pull request merged**